pkg/pillar/hypervisor/kvm.go

```diff
@@ -25,6 +25,7 @@ import (
 // KVMHypervisorName is a name of kvm hypervisor
 const KVMHypervisorName = "kvm"
 const minUringKernelTag = uint64((5 << 16) | (4 << 8) | (72 << 0))
+const swtpmTimeout = 5 // seconds
```

Comment:
Do we know how long time this takes on the VMs running the eden tests? Maybe we will need a longer timeout.

Reply:
There are a mainly two factors that can be hard to time exactly, there is system load (spawning a process) and TPM response speed (getting the encryption key). I never had any problem with 5 seconds in eden tests, but I'll change it to 10 just in case.